### ISSUE TITLE
discussion

### ISSUE THREAD

**Issue by raedah** [jamesphillipturpin/joinmarket#2]
Looks like you have continued to make progress. Is it working well?

**Comment by jamesphillipturpin** [comment]
It meets my needs anyway - earns more coinjoin fees than I pay in
transaction fees, and mixes my coins too. And I don't have to second guess
my prices. If people pay more or less it will adjust.

**Comment by raedah** [comment]
If there are some pieces of this that are obvious improvements, with no downsides, and are finished being optimized by you, then you could consider submitting specific parts of functionality as a pull request to jm oscillator.

**Comment by jamesphillipturpin** [comment]
That sounds good. I think I should refactor it with oscillator such that
the default behavior is that of the current oscillator, but just one
comment "#" can be deleted to give the behavior I intend.

**Comment by raedah** [comment]
Are you available for paid work?

**Comment by jamesphillipturpin** [comment]
No, I'm increasingly busy with my unrelated day job (technical drawing for
building architecture, not computer programming) ever since the November
2016 election. I don't have my head in my own private computer programming
projects or Bitcoin anymore, much less the JoinMarket code base. So it's
probably a bad match for everyone involved.
